build.gradle

```
@@ -2,15 +2,21 @@ plugins {
     id 'com.palantir.docker' version '0.25.0'
     id 'org.jetbrains.kotlin.jvm' version '1.6.21'
     id 'application'
-    id "org.owasp.dependencycheck" version "7.4.4"
+    id "org.owasp.dependencycheck" version "8.2.1"
```

```suggestion
    id "org.owasp.dependencycheck" version "8.3.1"
    id "com.gorylenko.gradle-git-properties" version "2.4.1"
```
The last plugins is used for https://exactpro.atlassian.net/browse/TH2-4987